src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3667,6 +3767,28 @@ private static ImmutableArray<Symbol> FixPartialMember(ImmutableArray<Symbol> sy
             return Remove(symbols, implementation);
         }
 
+        // PROTOTYPE(partial-properties): is there some abstraction that would make this nice?
+        private static ImmutableArray<Symbol> FixPartialMember(ImmutableArray<Symbol> symbols, SourcePropertySymbol part1, SourcePropertySymbol part2)
```

Comment:
nit: Consider carrying over an xml doc like the one on previous FixPartialMember overload

Reply:
I'm going to condense the doc as I feel the existing doc has excessive level of detail.